src/App.js

```diff
@@ -8,22 +8,24 @@ import users from './api/users';
 
 const App = () => (
   <div className="App">
-    <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
+    <div className="container">
+      <h1>Static list of posts</h1>
+      <p>
+        <span>posts: </span>
+        {posts.length}
+      </p>
 
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
+      <p>
+        <span>comments: </span>
+        {comments.length}
+      </p>
 
-    <p>
-      <span>Users: </span>
-      {users.length}
-    </p>
+      <p>
+        <span>Users: </span>
+        {users.length}
+      </p>
+      <PostList postsData={posts} usersData={users} commentsData={comments} />
```

Comment:
make a variable Prepared posts and pass only one prop to this component

Reply:
Fixed!